simulation/modules/plotpoint.py

```diff
@@ -0,0 +1,72 @@
+import attr
+import matplotlib.pyplot as plt
+import numpy as np
+
+from simulation.module import Module, ModuleState
+from simulation.state import State
+
+
+@attr.s(kw_only=True)
+class PlotPointState(ModuleState):
+    step_num: int = attr.ib(default=0)
+    fungal_burdens: np.ndarray = attr.ib(factory=lambda: np.zeros(1000, dtype=int))
```

Comment:
Could you extract this `1000` to just be a file-scope constant variable, just so we don't repeat it 4 times?